packages/destination-actions/src/destinations/mixpanel/groupIdentifyUser/__tests__/snapshot.test.ts

```diff
@@ -0,0 +1,75 @@
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import { generateTestData } from '../../../../lib/test-data'
+import destination from '../../index'
+import nock from 'nock'
+
+const testDestination = createTestIntegration(destination)
+const actionSlug = 'groupIdentifyUser'
+const destinationSlug = 'Mixpanel'
+const seedName = `${destinationSlug}#${actionSlug}`
+
+describe(`Testing snapshot for ${destinationSlug}\'s ${actionSlug} destination action:`, () => {
```

Comment:
```suggestion
describe(`Testing snapshot for ${destinationSlug}'s ${actionSlug} destination action:`, () => {
```
The backslash is causing the lint test to fail.